client/src/index.jsx

```diff
@@ -29,7 +31,7 @@ class App extends React.Component {
     let listingID = url.split('/')[3];
 
     $.ajax({
-      url: `http://localhost:3000/${listingID}/reviews`,
+      url: `http://localhost:3006/${listingID}/reviews`,
```

Comment:
this looks good for now but i think this is where we will need to use relative urls instead of absolute. im still working out the kinks on my proxy so i could be wrong 🙂